src/main/java/movies/spring/data/neo4j/movies/MovieService.java

```diff
@@ -58,6 +59,18 @@ public MovieDetailsDto fetchDetailsByTitle(String title) {
 				.orElse(null);
 	}
 
+	public int voteInMovieByTitle(String title) {
+		return this.neo4jClient
+				.query( "MATCH (m:Movie {title: $title}) " +
+						"WITH m, (CASE WHEN exists(m.votes) THEN m.votes ELSE 0 END) AS currentVotes " +
+						"SET m.votes = currentVotes + 1;" )
+				.in( database() )
```

Comment:
that will fail with 3.5

Reply:
I will check how the other services handles it.